cmd/conformance/main.go

```diff
@@ -324,10 +326,22 @@ func main() {
 			log.Fatal(err)
 		}
 
-		// Load artifact
-		file, err := os.Open(os.Args[len(os.Args)-1])
-		if err != nil {
-			log.Fatal(err)
+		var artifactPolicyOption verify.ArtifactPolicyOption
+		fileOrDigest := os.Args[len(os.Args)-1]
+
+		// Load digest or file
+		if strings.HasPrefix(fileOrDigest, "sha256:") {
+			digest, err := hex.DecodeString(fileOrDigest[7:])
+			if err != nil {
+				log.Fatal(err)
+			}
+			artifactPolicyOption = verify.WithArtifactDigest("sha256", digest)
+		} else {
```

Comment:
Should we fail if there is a `:` but a different prefix? In case we switch to sha512 at some point, we can fail gracefully

Reply:
It was just as easy to add support for any digest algorithm (sigstore-go will error gracefully if the digest algorithm is unsupported)